frontend/tests/console/utils/SecurityPage.ts

```diff
@@ -0,0 +1,88 @@
+/** biome-ignore-all lint/performance/useTopLevelRegex: this is a test */
```

Comment:
Maybe we exclude the biome config from running against these playwright test files